### ISSUE TITLE
Python3 UniCode Error when opening README.rst

### ISSUE THREAD

**Issue by kr-stn** [shichao-an/homura#5]
When installing under Python 3.5 with `python setup.py install` the setup fails with the error message

``` python
Traceback (most recent call last):
  File "setup.py", line 8, in <module>
    long_description=open('README.rst').read(),
  File "C:\Program Files\Python35\lib\encodings\cp1252.py", line 23, in decode
    return codecs.charmap_decode(input,self.errors,decoding_table)[0]
UnicodeDecodeError: 'charmap' codec can't decode byte 0x81 in position 48: character maps to <undefined>
```

I get the same error when installing through `pip install homura`.

I suspect this is caused by `README.rst` beeing encoded in UTF8.

I could solve this by changing line 8 to

``` python
long_description=open('README.rst', encoding="utf8").read()
```

This however would break compatability with Python2.

**Comment by kr-stn** [comment]
A possible fix would be to change the `setup.py` like #6 Now the install on my machine works with Python2 and Python3.

**Comment by shichao-an** [comment]
Thanks. Already merged #6
